packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/specialNumberForm/special-number-form.html

```diff
@@ -1,124 +1,142 @@
+            data-translate="telephony_order_identity_first_name"
         ></label>
-        <p
-            data-ng-bind="$ctrl.wallet.representative.lastname"
-            name="lastname"
-        ></p>
+        <input
+            type="text"
+            class="oui-input"
+            name="firstName"
+            id="firstName"
+            data-ng-model="$ctrl.ngModel.firstname"
+            data-required
+        />
     </div>
     <div class="form-group">
         <label
-            data-translate="telephony_order_identity_first_name"
-            for="firstname"
+            for="lastName"
+            data-translate="telephony_order_identity_name"
         ></label>
-        <p
-            data-ng-bind="$ctrl.wallet.representative.firstname"
-            name="firstname"
-        ></p>
+        <input
+            type="text"
+            class="oui-input"
+            name="lastName"
+            id="lastName"
+            data-ng-model="$ctrl.ngModel.name"
+            data-required
+        />
     </div>
     <div class="form-group">
         <label
-            data-translate="telephony_order_identity_email"
             for="email"
+            data-translate="telephony_order_identity_email"
         ></label>
-        <p data-ng-bind="$ctrl.wallet.representative.email" name="email"></p>
+        <input
+            type="text"
+            class="oui-input"
+            name="lastName"
+            id="lastName"
+            data-ng-model="$ctrl.ngModel.email"
+            data-required
+        />
     </div>
     <div class="form-group">
         <label
-            data-translate="telephony_order_coordinate_legalform_company"
-            for="company"
+            data-translate="telephony_billingAccount_svaWallet_kyc_identity_company_kind"
+            for="companyKind"
         ></label>
-        <p data-ng-bind="$ctrl.wallet.company.name" name="company"></p>
+        <oui-select
+            items="$ctrl.companyKindsValues"
+            match="name"
+            name="companyKind"
+            model="$ctrl.companyKind"
+            on-change="$ctrl.onCompanyKindChange(modelValue)"
+            searchable
+            required
+        >
+        </oui-select>
     </div>
+
     <div class="form-group">
         <label
             data-translate="telephony_order_identity_siret"
             for="siret"
         ></label>
-        <p data-ng-bind="$ctrl.wallet.company.identificationNumber"></p>
+        <input
+            type="text"
+            class="oui-input"
+            name="siret"
+            data-ng-model="$ctrl.ngModel.siret"
+            data-required
+        />
     </div>
-    <div
-        data-ng-if="$ctrl.wallet.representative.addressOfResidence.streetType"
-        class="form-group"
-    >
+    <div class="form-group">
         <label
             data-translate="telephony_order_address_street_type"
             for="street-type"
         ></label>
-        <p
+        <input
```

Comment:
**nitpick(blocking):** Can you link `label`s and `input`s via `for=""` and `id=""` attributes?

Reply:
Yes